barber-compiler/build.gradle

```diff
@@ -5,6 +5,7 @@ sourceCompatibility = rootProject.ext.sourceCompatibilityVersion
 targetCompatibility = rootProject.ext.targetCompatibilityVersion
 
 dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Right! Or you could do it simpler by just matching the simple name too, like how Butter Knife does with Nullable.

See:
https://github.com/JakeWharton/butterknife/blob/master/butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeProcessor.java#L1169-L1181